app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -114,6 +115,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         viewBinding.transparentBackground.setOnClickListener(v -> finish());
         viewBinding.card.setOnClickListener(null);
+        viewBinding.autoDownloadCheckBox.setChecked(UserPreferences.isEnableAutodownload());
```

Comment:
Is this needed? I think the checkbox is only displayed if auto download is enabled, anyway.